corporate/lib/stripe.py

```diff
@@ -3162,8 +3171,16 @@ def approve_sponsorship(self) -> str:
             self.write_to_audit_log(
                 event_type=AuditLogEventType.SPONSORSHIP_APPROVED, event_time=timezone_now()
             )
-        # TODO: Add something (probably email) to let remote server
-        # organization know that the sponsorship request was approved.
+        send_email(
+            "zerver/emails/sponsorship_approved_community_plan",
+            to_emails=[self.remote_server.contact_email],
+            from_address=FromAddress.SUPPORT,
```

Comment:
You can hardcode `sales@zulip.com` here; it's already hardcoded in ~25 places in `corporate/`. Would not be a bad idea to make it a setting though as a follow-up.